mongodb/spring-cloud-starter-stream-source-mongodb/src/main/java/org/springframework/cloud/stream/app/mongodb/source/MongoDBSourceProperties.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2016 the original author or authors.
```

Comment:
Copyright hasn't been updated.
Not a big deal and we can address on merge, but would be good to follow with our requirements for future contributions.